recipe/meta.yaml

```diff
@@ -1,32 +1,30 @@
-{% set name = "pandas_market_calendars" %}
-{% set version = "1.7" %}
-{% set sha256 = "706e5a9fac3e7722fa5875a230a0abc813128ffff3f36064da676b445017575d" %}
+{% set name = "pandas-market-calendars" %}
```

Comment:
@yehoshuadimarsky just for future reference, conda does not treat `-` and `_` as equivalent like pypa does, so this change actually makes a separate condo-forge package:

https://anaconda.org/conda-forge/pandas-market-calendars
vs 
https://anaconda.org/conda-forge/pandas_market_calendars